kakao/src/main/kotlin/com/agoda/kakao/picker/date/KDatePickerDialog.kt

```diff
@@ -0,0 +1,22 @@
+package com.agoda.kakao.picker.date
+
+import android.app.DatePickerDialog
+import android.widget.DatePicker
+import androidx.appcompat.widget.AlertDialogLayout
+import com.agoda.kakao.common.views.KBaseView
+import com.agoda.kakao.text.KButton
+
+/**
+ * View for interact with default date picker dialog
+ *
+ * @see DatePickerDialog
+ */
+class KDatePickerDialog : KBaseView<KDatePickerDialog>({ isInstanceOf(AlertDialogLayout::class.java) }) {
+    val datePicker = KDatePicker { isInstanceOf(DatePicker::class.java) }
+    val okButton = KButton {
```

Comment:
is `okButton` and `cancelButton` descendants of the `DatePicker` widget? If so, please add `isDescendantOfA { isInstanceOf(...) }` to the matcher.